src/App.tsx

```diff
@@ -8,35 +8,85 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import * as clientService from './api/posts';
+import { Post } from './types/Post';
+import { User } from './types/User';
+import { getUsers } from './api/users';
 
 export const App: React.FC = () => {
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [users, setUser] = useState<User[]>([]);
+  const [userId, setUserId] = useState<number>(0);
+  const [error, setError] = useState('');
+  const [loader, setLoader] = useState(false);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isLoading, setIsLoading] = useState(false);
```

Reply:
About avoid adding empty comment (whitespaces). In the demo app, we can add whitespaces, and in task don`t ask to do it. So maybe need change it in demo and in task. Becouse for now, seem like need to give opportunity add whitespaces